### PR TITLE
fixed IDEA inspection warnings

### PR DESCRIPTION
**Pull request by firewave**

### CODE REVIEW

**Comment by johnthagen** [comment]
@firewave Can you please provide example output that this fails previously that this fixes?

@SJ-Innovation Please review.

**Comment by firewave** [comment]
There are no failures. It's just the inspection that IntelliJ IDEA highlighting shows - it's basically just some cleanups.

**Comment by johnthagen** [comment]
Okay, if you can confirm everything still works as expected, I can merge this in.

File: src/com/github/johnthagen/cppcheck/CppcheckInspection.java [hunk 1]
```diff
@@ -114,7 +114,7 @@ private static String prependIncludeDir(@NotNull String cppcheckOptions, @NotNul
     // Example output line:
     // [C:\Users\John Hagen\ClionProjects\test\main.cpp:12]: (style) Variable 'a' is not assigned a value.
     // [main.cpp:12] -> [main.cpp:14]: (performance) Variable 'a' is reassigned a value before the old one has been used.
-    Pattern pattern = Pattern.compile("^\\[(.+?):(\\d+)\\](?:\\s+->\\s+\\[.+\\])?:\\s+\\((\\w+)\\)\\s+(.+)");
+    Pattern pattern = Pattern.compile("^\\[(.+?):(\\d+)](?:\\s+->\\s+\\[.+])?:\\s+\\((\\w+)\\)\\s+(.+)");
```
Comment (johnthagen) [inline src/com/github/johnthagen/cppcheck/CppcheckInspection.java:117]: Need to ensure that this doesn't break the regex parsing.

Comment (SJ-Innovation) [reply src/com/github/johnthagen/cppcheck/CppcheckInspection.java:117]: Can confirm that this does not seem to break the parsing of the regex

Comment (firewave) [reply src/com/github/johnthagen/cppcheck/CppcheckInspection.java:117]: before:
https://regex101.com/r/GO7EIs/1
after:
https://regex101.com/r/EQPzC6/1

**Comment by johnthagen** [comment]
Merged, thanks @firewave!

**Pull request merged**